src/org/voltdb/jsr107/CacheEventConsumer.java

```diff
@@ -63,7 +65,7 @@ public class CacheEventConsumer implements Runnable {
     /**
      * Keep running until told to stop..
      */
-    boolean keepGoing = true;
+    AtomicBoolean keepGoing = new AtomicBoolean(true);
```

Comment:
Since this is read and written by different threads, I think we should use `AtomicBoolean`